app/backend/src/PresentationRulesEditorRpcImpl.ts

```diff
@@ -25,10 +24,20 @@ export class PresentationRulesEditorRpcImpl extends PresentationRulesEditorRpcIn
       .filter((entry) => entry.isFile() && (entry.name.endsWith(".ibim") || entry.name.endsWith(".bim")))
       .map((file) => {
         const entry = fs.statSync(path.join(dir, file.name));
-        return { name: file.name, dateModified: entry.mtime, size: prettyBytes(entry.size) };
+        return { name: file.name, dateModified: entry.mtime, size: this.formatBytes(entry.size) };
       });
   }
 
+  private formatBytes(bytes: number, decimals = 2) {
```

Comment:
Looks like this is unused.

```suggestion
  private formatBytes(bytes: number) {
```